### ISSUE TITLE
Splash not found: always black screen. Cordova 6, ios

### ISSUE THREAD

**Issue by firepol** [devgeeks/PrivacyScreenPlugin#11]
Hi there,

I just tested this with cordova 6.0.0 and the screen is always black.

To reproduce, install cordova 6.0.0 (the latest version), then:
```
cordova create Test
cd Test
cordova platform add ios
cordova plugin add cordova-plugin-privacyscreen
cordova emulate ios --target=iPhone-6-Plus
```

Cordova creates the HelloCordova app, which has (inside of the Xcode Project Resources) an Image.xcassets catalog including AppIcon and LaunchImage.
The image names in the LaunchImage catalog are standard: Default.png, Default-736h.png etc.

So I was expecting, leaving as is, that the privacy screen plugin would find the default*.png files and use one of them for the privacy screen, instead of the black screen, but it seems it can't find the images.

I tried to edit the HelloCordova-info.plist file and add the following:
```
    <key>UILaunchImageFile</key>
    <string>LaunchImage</string>
```

I've put LaunchImage, as that's the name of the image set. I tried to put Default.png, I tried Default. I tried add an image called Splash.png directly in the Resources folder of the Xcode project and tried to use Splash.png, or Splash.

When I press 2 times the "Home" button (I actually do that in the simulator by pressing Cmd+Shit+H 2 times), or when I resume the app after putting it in background, I always get the black screen instead of the splash image I was expecting.

I tried also to modify the plugin .m file and in the GetImageName to return "LaunchImage" or "Splash" but when I minimize the app and check it, it always has a black screen and not the splash screen.

It would be nice if you test this and maybe tell what I'm doing wrong or find a fix, thank you.

**Comment by devgeeks** [comment]
Yeah, the problem is that the plugin looks for images in the main bundle (like the old Resources folder) but probably cannot see the images in the new Xcassets. 

I think much like #10, this calls for a change in the operation of the plugin so that it no longer relies on the splash screen images.

Something like a blur or a generic image that can be overridden in settings or something.

**Comment by firepol** [comment]
A generic blur or generic image is a good idea, but it would be nice to still have the possibility to specify an image/splash screen. So if possible, improving the lookup logic would be ideal.

**Comment by devgeeks** [comment]
Basically, it needs to add support for the new xcasset images if possible.

**Comment by okonon** [comment]
@devgeeks i am wondering if you had a chance to work on this issue?

**Comment by devgeeks** [comment]
Not yet, I have some ideas. I might look at it this weekend.

**Comment by mattalco** [comment]
I was also curious to how this is progressing. Thanks :)

**Comment by lifeofcoding** [comment]
The company I work for is making a HIPAA compliant hybrid application for iOS, and this plugin is perfect for it, but the client absolutely hates that it is just a black screen. The app has such a nice splash image too </3

**Comment by devgeeks** [comment]
It's certainly possible now to still have the splashscreen show... it just takes some jiggery pokery until I get time to figure it out.

The catch is that newer cordova-ios uses the xcassets instead 

of just a folder in the Resources group.

It's a hack workaround til I can update the plugin.

If you manually add your splash screens to your Xcode project where they used to be (as well as the actual ones in use in the xcassets), you should see the splashscreen again using this plugin.

    open -a Xcode platforms/ios/

Add a group/folder under Resources calls "splash" and right click it and add / copy your splash images there.

<img width="343" alt="screen shot 2016-03-31 at 7 45 58 am" src="https://cloud.githubusercontent.com/assets/554999/14157031/111a77a0-f715-11e5-9200-0ba566ef2a02.png">

<img width="261" alt="screen shot 2016-03-31 at 7 46 19 am" src="https://cloud.githubusercontent.com/assets/554999/14157073/387dfe0c-f715-11e5-9ded-8b76baaed6d0.png">

![screen shot 2 2016-03-31 at 7 46 34 am](https://cloud.githubusercontent.com/assets/554999/14157275/0627527c-f716-11e5-9039-118012025cae.png)

Add the splashscreen images from the `Images.xcassets/Launchimage.launchimage/` folder:

<img width="941" alt="screen shot 2016-03-31 at 7 47 14 am" src="https://cloud.githubusercontent.com/assets/554999/14157141/7474b8ec-f715-11e5-9834-9b1140d71160.png">

This also doesn't double up on the images, it just kind of links them into the additional folder/group.

**Comment by yyfearth** [comment]
@devgeeks Seems like in Cordova 6.1.1 with Cordova iOS 4.1.1, the workaround not working either.

**Comment by devgeeks** [comment]
I'll try it out.

**Comment by Alex-Sessler** [comment]
I added a PR that likely fixes this: https://github.com/devgeeks/PrivacyScreenPlugin/pull/16

**Comment by mattalco** [comment]
@devgeeks If @Alex-Sessler's fix works for you, do you know when you might have time to merge this?

**Comment by devgeeks** [comment]
I am going to put aside some time to look at PRs today. Hopefully merge everything in.

**Comment by devgeeks** [comment]
PR merged. This seems to be fixed now @0.3.0

Everyone try the latest version of the plugin. If it's still broken we can re-open, but it seems fixed to me.
